cms/conf.py

```diff
@@ -110,6 +110,10 @@ def __init__(self):
         self.admin_listen_port = 8889
         self.admin_cookie_duration = 10 * 60 * 60  # 10 hours
 
+        # APIWebServer.
+        self.api_listen_address = ""
```

Comment:
In order to keep the change in the main code, can we have these two as arguments to the script instead?